diktat-analysis.yml

```diff
@@ -452,7 +452,7 @@
   enabled: true
 # There's a property in KDoc which is already present
 - name: KDOC_DUPLICATE_PROPERTY
-  enabled: false
+  enabled: true
```

Comment:
can we add an unit test for it?

Reply:
What do you mean?

Reply:
a junit test with code example like:
```kotlin
/**
 * @property a
 * @property a
 */
fun foo(a: String) {
    // do something
}
```

And it expects a warn about duplication in param/property

Reply:
ohh...missed tests.. sorry